src/modules/home/HomePage.js

```diff
@@ -15,15 +16,21 @@ import FontAwesome from 'react-fontawesome';
 import './index.css';
 import Config from '../../config';
 import { hashHistory } from 'react-router';
-import appSession$ from './../../modules/app/appSession';
 import Logos from '../../components/logos/Logos';
 
-const reportURL = Config.magasin.urls.client.report.goToReport(appSession$);
-const magasinURL = Config.magasin.urls.client.magasin.goToMagasin(appSession$);
-const analysisURL = Config.magasin.urls.client.analysis.addAnalysis(appSession$);
+const reportURL = s => Config.magasin.urls.client.report.goToReport(s);
```

Comment:
appSession$ did not worked so I have to send session some parameter. We needed the session to show Museum id and Collection id on URL.